gallery/src/styles/NxTile/NxTileWithActionsExample.tsx

```diff
@@ -8,26 +8,20 @@ import React from 'react';
 
 const NxTileWithActionsExample = () =>
   <div className="nx-tile">
-    <div className="nx-tile__actions">
-      <button className="nx-btn nx-btn--tertiary">Action 1</button>
-      <button className="nx-btn nx-btn--tertiary">Action 2</button>
-    </div>
     <div className="nx-tile-header">
-      <div className="nx-tile-header__title"><h2 className="nx-h2">NX Tile with Actions</h2></div>
+      <div className="nx-tile-header__title">
+        <h2 className="nx-h2">NX Tile with Actions - and this line is meant to wrap so that we can test truncation</h2>
+      </div>
+      <div className="nx-tile-header__subtitle">The Subtitle</div>
```

Comment:
Would probably be good to have a longer subtitle in one example to prove they wrap

Reply:
> Would probably be good to have a longer subtitle in one example to prove they wrap

Added to the subtitle example: https://github.com/sonatype/sonatype-react-shared-components/pull/34/commits/0d4cc745e574428f292fb47a85f4fc77c75b41cf